doc/whats-new.rst

```diff
@@ -32,6 +32,10 @@ New Features
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- ``Datatree``'s ``as_array`` renamed ``to_dataarray`` to align with ``Dataset``. (:pull:`8789`)
```

Comment:
@TomNicholas should this have been kept out of breaking changes? mostly because it's not actually released? Wasn't sure where I should keep it. [reference](https://github.com/flamingbear/xarray/pull/6#discussion_r1505017764)

Reply:
https://github.com/pydata/xarray/issues/8807 and reverted. 😬